locale/en.po

```diff
@@ -4015,7 +4015,7 @@ msgid "ORGANELLES_BUTTON"
 msgstr "Organelles"
 
 msgid "ORGANELLES_WILL_BE_UNLOCKED_NEXT_GENERATION"
-msgstr "Unlock conditions are checked only once per generation when entering the editor"
+msgstr "Unlock conditions are checked only once per generation when entering the editor. Some organelles can alternatively be unlocked through endosymbiosis"
```

Comment:
```suggestion
msgstr "Unlock conditions are checked only once per generation when entering the editor. Some organelles can alternatively be unlocked through endosymbiosis."
```

For consistency I'd end the second sentence with a dot as well.

With that change I think this might just be fully done, which is probably why I marked this as an easy issue as it doesn't take much to do.